<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocationForm</class>
 <widget class="QWidget" name="LocationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#btn_apply,#pushButton_del{
   color:white;
	background-color: rgb(78, 154, 6);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>latuitude:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>valid:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit_altitude"/>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>angle:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>hemisphere:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>altitude:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLineEdit" name="lineEdit_hemisphere"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>speed:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="lineEdit_satNumber"/>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="lineEdit_angle"/>
     </item>
     <item row="1" column="7">
      <widget class="QLineEdit" name="lineEdit_timestamp"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_longtuitude"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit_speed"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_latuitude"/>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>timestamp:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_valid">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>satNumber:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>longtuitude:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_del">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
